bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -120,21 +122,16 @@
     performWarmup();
   }
 
-  private static SslContextBuilder createGrpcSslBuilder() {
-    SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
-    sslBuilder.ciphers(null);
-    return sslBuilder;
-  }
-
-  private static synchronized SslContextBuilder getSslBuilder() {
+  private synchronized static SslContext createSslContext() throws SSLException {
     if (sslBuilder == null) {
-      sslBuilder = createGrpcSslBuilder();
+      sslBuilder = GrpcSslContexts.forClient().ciphers(null);
+      if (OpenSsl.isAvailable()) {
+        LOG.info("gRPC is using the OpenSSL provider");
+      } else {
+        LOG.info("gRPC is using the JDK provder");
```

Comment:
is that when JDK provider is used, we need to check org.eclipse.jetty.alpn.ALPN ?